src/config/env.js

```diff
@@ -1,22 +1,27 @@
 require("dotenv-flow").config();
 
-module.exports = Object.freeze({
+const config = Object.freeze({
     // Database
     db_host: process.env.DB_HOST,
     db_port: process.env.DB_PORT,
-    db_uri: process.env.DB_URI,
+    db_name: process.env.DB_NAME,
+    db_uri: process.env.DB_URI ||
+    (
+        process.env.DB_HOST
+            && process.env.DB_PORT
+            && process.env.DB_NAME
+            && `mongodb://${process.env.DB_HOST}:${process.env.DB_PORT}/${process.env.DB_NAME}`
+    ),
     db_user: process.env.DB_USER,
     db_pass: process.env.DB_PASS,
-    db_name: process.env.DB_NAME,
 
+    // App
     session_secret: process.env.SESSION_SECRET,
-
     port: process.env.PORT,
-
     god_token: process.env.GOD_TOKEN,
-
     test_log_requests: JSON.parse(process.env.TEST_LOG_REQUESTS),
-
     admin_email: process.env.ADMIN_EMAIL,
     admin_password: process.env.ADMIN_PASSWORD,
 });
+
+module.exports = config;
```

Comment:
Rationale for this change? I liked the `module.exports =` approach and don't really see reasons to change

Reply:
I'm actually not sure why it changed, maybe I was exporting additional things and changed the structure, but it can be reverted.